README.md

```diff
@@ -1,10 +1,32 @@
 # automate
-Created with Electron JS and integrating with both Azure DevOps and GitHub, automate allows you to automate tasks such as changing branches across enterprise workspaces and solutions, tagging multiple repositories at the same time, and much more.
+automate is an Electron JS based source management program with Azure DevOps and Github integration. automate focuses on streamlining workflow tasks such as changing branches across enterprise workspaces and solutions, as well as tagging multiple repositories at the same time.
+
 
  [![Screenshot of the landing page.](images/landing.png)]()
 
-## Getting Started
-Make sure that you have [Node.js](https://nodejs.org/en/) installed. The minimum required version is 14.
+## Features
+* **Branch management:** automate branch management across enterprise workspaces and solutions to organize your workflow 
+* **Tagging multiple repositories:** tag multiple repositories simultaneously to optimize your performance
+* **User-oriented:** planned real-time theme customization for a user experience that fits your personal needs
+
+
+## Installation
+1. Install [Node.js](http://nodejs.org/en/) - the recommended version is 14.0.0 or above
+2. Download automate files and extract them to a new folder
+3. Open command prompt and navigate to the root directory for automate
+4. Run `npm install` to install local dependencies
+5. Run `npm start` to launch the program
+
+
+## Contribute
```

Comment:
User and dev friendly contribution tab to encourage any and all user feedback on automate